_sass/_post.scss

```diff
@@ -30,3 +30,7 @@
   }
 
 }
+
+p img {
```

Comment:
Sure thing.

Does it still need to be wrapped by the `<p>` tag? Might be redundant at this point, after moving it into the appropriate selector.